### PR TITLE
Create Thai_Spices_Aromatic_Ginger.py

### PR DESCRIPTION
**Pull request by ANUGAMAGE**
Going to add files

### CODE REVIEW

**Comment by Sulstice** [comment]
Can you confirm everything looks alright to merge in?

**Comment by ANUGAMAGE** [comment]
Sure sul

File: README.md
Comment (ANUGAMAGE) [inline README.md:1]: GlobalChem API-----AttributeError: type object 'MangoPhytocompounds' has no attribute 'get_smiles'
Fair softw error

**Review by ANUGAMAGE — COMMENTED** [review]
Need to find solutions for these errors
GlobalChem API-----AttributeError: type object 'MangoPhytocompounds' has no attribute 'get_smiles'
Fair softw error

**Pull request merged**